### ISSUE TITLE
Crash on Samsung devices running Android 4.2 

### ISSUE THREAD

**Issue by blinskey** [blinskey/greek-reference#105]
v1.7.0
Android 4.2
Device: wiko

```
java.lang.NoClassDefFoundError: android.support.v7.internal.view.menu.MenuBuilder
at android.support.v7.app.ActionBarActivityDelegateBase.initializePanelMenu(Unknown Source)
at android.support.v7.app.ActionBarActivityDelegateBase.preparePanel(Unknown Source)
at android.support.v7.app.ActionBarActivityDelegateBase.doInvalidatePanelMenu(Unknown Source)
at android.support.v7.app.ActionBarActivityDelegateBase.access$100(Unknown Source)
at android.support.v7.app.ActionBarActivityDelegateBase$1.run(Unknown Source)
at android.os.Handler.handleCallback(Handler.java:800)
at android.os.Handler.dispatchMessage(Handler.java:100)
at android.os.Looper.loop(Looper.java:194)
at android.app.ActivityThread.main(ActivityThread.java:5391)
at java.lang.reflect.Method.invokeNative(Native Method)
at java.lang.reflect.Method.invoke(Method.java:525)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:833)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:600)
at dalvik.system.NativeStart.main(Native Method)
```

**Comment by blinskey** [comment]
This appears to be a known bug on certain devices running Android 4.2. See [Android issue 78377](https://code.google.com/p/android/issues/detail?id=78377), which includes [a workaround in the comments](https://code.google.com/p/android/issues/detail?id=78377#c138).

**Comment by blinskey** [comment]
New report:

Application version: 1.8.0
Android version: Android 4.2
Device: P9

```
java.lang.NoClassDefFoundError: android.support.v7.internal.view.menu.MenuBuilder
at android.support.v7.app.ActionBarActivityDelegateBase.initializePanelMenu(Unknown Source)
at android.support.v7.app.ActionBarActivityDelegateBase.preparePanel(Unknown Source)
at android.support.v7.app.ActionBarActivityDelegateBase.doInvalidatePanelMenu(Unknown Source)
at android.support.v7.app.ActionBarActivityDelegateBase.access$100(Unknown Source)
at android.support.v7.app.ActionBarActivityDelegateBase$1.run(Unknown Source)
at android.os.Handler.handleCallback(Handler.java:725)
at android.os.Handler.dispatchMessage(Handler.java:92)
at android.os.Looper.loop(Looper.java:153)
at android.app.ActivityThread.main(ActivityThread.java:5293)
at java.lang.reflect.Method.invokeNative(Native Method)
at java.lang.reflect.Method.invoke(Method.java:511)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:833)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:600)
at dalvik.system.NativeStart.main(Native Method)
```

**Comment by blinskey** [comment]
Assuming that this is still ProGuard-related, this would have been fixed with the removal of obfuscation in v1.9.1.

**Comment by blinskey** [comment]
Correction to previous comment: This is present in v1.9.2, and it looks like obfuscation may actually be the only way to fix it. If we ever reenable ProGuard, we could implement the fix at that time. For now, however, this is a low-priority issue.

StackOverflow discussion: http://stackoverflow.com/questions/24809580/noclassdeffounderror-android-support-v7-internal-view-menu-menubuilder

**Comment by blinskey** [comment]
Planning to fix this in v1.9.3 via ProGuard obfuscation. Need to ensure that we set up ProGuard so that useful stack traces are still created.
